web/src/utils/dashboard/convertMapsData.ts

```
@@ -0,0 +1,139 @@
+  }
+
+  const options: any = {};
+
+  options.tooltip = {
+    trigger: "item",
+    showDelay: 0,
+    transitionDuration: 0.2,
+    backgroundColor: "rgba(255,255,255,0.8)",
+    formatter: function (params: any) {
+      let formattedValue = params.value;
+      if (formattedValue === "-" || isNaN(formattedValue)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)